social_core/backends/oauth.py

```diff
@@ -247,7 +247,8 @@ def request_token_extra_arguments(self):
     def unauthorized_token(self):
         """Return request for unauthorized token (first stage)"""
         params = self.request_token_extra_arguments()
-        params.update(self.get_scope_argument())
+        if self.name not in ['trello']:
+            params.update(self.get_scope_argument())
```

Comment:
If this is really needed, overriding `get_scope_argument` in `TrelloOAuth` is the way to go.

Reply:
Trello REQUEST_TOKEN_URL (https://trello.com/1/OAuthGetRequestToken) does not take any arguments.

If we send the scope here then the URL looks like this **https://trello.com/1/OAuthGetRequestToken?scope=read,write**

As per above mentioned Trello request does not take any arguments. Trello gives an error like this...

ERROR: 400 Client Error: Bad Request for URL: https://trello.com/1/OAuthGetRequestToken?scope=read%2Cwrite

Reply:
Thanks, this should go as a comment to the code. And the implementation should override `get_scope_argument` method in `TrelloOAuth` instead of doing changes in generic OAuth backend.